client/src/pages/ProfileDetails/BookingForm/BookingForm.tsx

```diff
@@ -35,10 +37,30 @@ const BookingForm = (): JSX.Element => {
       value: 23,
     },
   ]);
+  const [rating, setRating] = useState(5);
 
   useEffect(() => {
     if (id) {
-      fetchProfile(id).then((data) => setAvailableTime(data.success.profile.availableTime));
+      fetchProfile(id).then((data) => {
```

Comment:
Aren't we already fetching the sitter profile within the Profile Details page? We can just pass the `availableTime` as a prop to the booking form :)

Reply:
I tried to change this but I was having issues with it and the previous code was fully functional so I just kept it so I could show it off at the presentation.